internal/bundler/snapshots/snapshots_default.txt

```diff
@@ -2006,9 +2006,9 @@ var require_es6_ns_export_async_function = __commonJS((exports) => {
 var require_es6_ns_export_enum = __commonJS((exports) => {
   var ns2;
   (function(ns3) {
-    let Foo3;
-    (function(Foo4) {
-    })(Foo3 = ns3.Foo || (ns3.Foo = {}));
+    let Foo2;
```

Comment:
I presume that the numbering in this snapshot is changed because the `const enum` was not emitted